ocs_ci/utility/kms.py

```diff
@@ -123,13 +126,22 @@ def __init__(self):
         )
         self.kmsid = None
         # Base64 encoded (with padding) token
-        self.vault_path_token = None
+        self.vault_path_token = Vault.vault_path_token
         self.vault_policy_name = None
         self.vault_kube_auth_path = "kubernetes"
         self.vault_kube_auth_role = constants.VAULT_KUBERNETES_AUTH_ROLE
         self.vault_kube_auth_namespace = None
         self.vault_cwd_kms_sa_name = constants.VAULT_CWD_KMS_SA_NAME
 
+    @staticmethod
+    def set_vault_token():
+        """
+        This is a static method fucntion that will set the token value.
+        """
+        if Vault.vault_path_token is None:
+            Vault.vault_path_token = Vault.generate_vault_token()
+        return Vault.vault_path_token
+
```

Comment:
I think we should use @classmethod here instead:

```suggestion
    @classmethod
    def set_vault_token(cls):
        """
        This is a class method that will set the token value.
        """
        if cls.vault_path_token is None:
            cls.vault_path_token = cls.generate_vault_token()
        return cls.vault_path_token

```

Reply:
Great idea Coady, made the changes.